index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <navbar>
```

Comment:
[OPTIONAL]
- [ ] As shown in the images below your positioning of the contents of the nav bar is not consistent with the provided figma image below, please kindly make the necessary adjustments so that your design closely matches the provided figma template :+1: 
- project design
![hh](https://github.com/fatmahussein/Green-Earth-Summit/assets/89055502/1d769c29-8e73-4edc-a257-135cf03dd829)
- your design
![yhh](https://github.com/fatmahussein/Green-Earth-Summit/assets/89055502/9772bd63-8329-4017-84c7-dc441d6f1b88)